src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
```

Comment:
You have an extra empty line after the package declaration. It is better to remove unnecessary empty lines for better code readability.